### PR TITLE
Add support for natural-language-processing

### PR DESCRIPTION
**Pull request by dmeijboom**
Within Facebook Messenger it's possible to enable natural-language-processing. You can simply add your wit.ai access token and Facebook will also send the NLP entities with the message.

Currently this isn't supported by this framework. With this commit I've implemented the NLP entities in the Message object from the Facebook Messenger API.

### CODE REVIEW

File: src/Model/Callback/Message.php [hunk 1]
```diff
@@ -101,14 +108,22 @@ public function hasQuickReply(): bool
     }
 
     /**
-     * @return array
-     */
+        * @return array
+        */
```
Comment (ker0x) [inline src/Model/Callback/Message.php]: Can you fix the indentation please !

File: src/Model/Callback/Message.php [hunk 2]
```diff
@@ -127,6 +142,13 @@ public static function create(array $callbackData)
         $quickReply = $callbackData['quick_reply']['payload'] ?? null;
         $attachments = $callbackData['attachments'] ?? [];
 
-        return new static($callbackData['mid'], $callbackData['seq'], $text, $quickReply, $attachments);
+        return new static(
+            $callbackData['mid'],
+            $callbackData['seq'],
+            isset($callbackData['nlp']) ? $callbackData['nlp']['entities'] : null,
```
Comment (ker0x) [inline src/Model/Callback/Message.php]: if `$callbackData['nlp']['entities']` doesn't exist, it should returns an `array`, not `null`. Can you also extract this logic inside a variable `$entities`, like `$quickReply` or `$attachments`

**Comment by dmeijboom** [comment]
Thanks for your feedback, I'll process it probably this week.

**Comment by AinCoder** [comment]
Hey, Buddy!
Can you explain how to add postback button on a message?
You just added it to persistent menus :(
Is there any URL for know more about this code? Like how to add postback button with a message

**Comment by ker0x** [comment]
Hello @AinCoder !

To add PostBack button on a message, you have to use the ButtonTemplate template, otherwise you can use quick replies !

**Comment by AinCoder** [comment]
@ker0x Yup, Got that after few hours :)
May I use this code for the commercial app?

**Comment by ker0x** [comment]
@AinCoder the library is realeased under the [MIT License](https://github.com/ker0x/messenger/blob/master/LICENSE) so yes you can use it for commercial purpose ;)

**Pull request closed without merging**